provers/sp1/driver/src/lib.rs

```diff
@@ -27,12 +31,14 @@ impl From<Sp1Response> for Proof {
 
 pub struct Sp1Prover;
 
+const SP1_PROVER_CODE: u8 = 1;
```

Comment:
should we have another place to distribute this enum to avoid conflict?? so far we have R0=3, SP1=1, etc.

Reply:
I might move the `ProofType` into `lib`